ibtk/include/ibtk/libmesh_utilities.h

```diff
@@ -514,14 +516,14 @@ void apply_transposed_constraint_matrix(const libMesh::DofMap& dof_map, libMesh:
  *
  * @seealso FEDataManager::updateQuadratureRule.
  */
-std::tuple<libMesh::ElemType, libMesh::QuadratureType, libMesh::Order>
-getQuadratureKey(const libMesh::QuadratureType quad_type,
-                 libMesh::Order order,
-                 const bool use_adaptive_quadrature,
-                 const double point_density,
-                 const libMesh::Elem* const elem,
-                 const boost::multi_array<double, 2>& X_node,
-                 const double dx_min);
+quadrature_key_type getQuadratureKey(libMesh::QuadratureType quad_type,
+                                     libMesh::Order order,
+                                     bool use_adaptive_quadrature,
+                                     double point_density,
+                                     bool allow_rules_with_negative_weights,
+                                     const libMesh::Elem* elem,
+                                     const boost::multi_array<double, 2>& X_node,
+                                     double dx_min);
```

Comment:
This is an incompatible change (so is the bounding box change) but it's really unlikely that any users will notice so I don't mind. We do need to update the documentation of this function.